### DISCUSSION TITLE
Dealing with Code lint

### DISCUSSION THREAD

**Discussion by sithlord48** [deskflow/deskflow#7536] [Development]
We should be linting our code and having contributors format it before hand this discussion is about how we should do this. 

Started from PR https://github.com/deskflow/deskflow/pull/7530 (decided not to merge)

Issue: https://github.com/deskflow/deskflow/issues/7527

**Comment by nbolton** [comment]
> TODO: Instead of commit ammdend and rebase (which GitHub doesn't really want you to do because of infinite loop bugs), we will add a comment to the PR instructing the author how to fix the problem.

**Reply by nbolton** [reply]
@sithlord48 I'll open a new PR based on this idea of adding a comment.

**Reply by nbolton** [reply]
WIP PR: https://github.com/deskflow/deskflow/pull/7539

**Comment by nbolton** [comment]
Examples of helpful generated comments (deleted when lint passes):
- https://github.com/deskflow/deskflow/pull/7539#issuecomment-2364102173
- https://github.com/deskflow/deskflow/pull/7539#issuecomment-2364102219
